README.md

```diff
@@ -25,3 +25,21 @@ and solve it as we did in the lectures. Give the final $\Theta$ complexity.
 
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. Add your answer to this markdown file.
+
+
+T(n) = 3T(n/3) + O(n)
+3T(n/3) represents the time complexity of recursing through the three thirds of the array each of roughly size (n/3).
+O(n) represent the time complexity of summing each of the elements of the subarrays.
```

Comment:
Aren't you just summing the partial results?

Reply:
Then it should really be T(n) = 3T(n/3) + O(1) like I had originally thought since the summation step is in constant time?